README.md

```diff
@@ -93,7 +93,7 @@ Run `make help` to see all the possible targets and variables:
     LEPTONICA_VERSION  Leptonica version. Default: 1.78.0
     TESSERACT_VERSION  Tesseract commit. Default: 4.1.0
     TESSDATA_REPO      Tesseract model repo to use. Default: _best
-    TESSDATA           Name of the traineddata file to start finetuning from (without file extension)
+    TESSDATA           Path to the .traineddata directory to start finetuning from. Default: './usr/share/tessdata'
```

Comment:
As the other descriptions don't use apostrophes, I suggest to omit them here, too.